include/teb_local_planner/robot_footprint_model.h

```diff
@@ -493,8 +491,46 @@ class LineRobotFootprint : public BaseRobotFootprintModel
     line_end_world.z = 0;
     marker.points.push_back(line_end_world);
 
-    marker.scale.x = 0.05; 
-    marker.color = color;
+    // footprint with min_obstacle_dist
+    markers.push_back(visualization_msgs::Marker());
+    visualization_msgs::Marker& marker2 = markers.back();
```

Comment:
S: better to make a copy and modify it:
- narrower lines
- different color
- z = 0.001 so it appears over the costmap